underfs/oss/src/main/java/alluxio/underfs/oss/OSSUnderFileSystem.java

```diff
@@ -226,6 +226,8 @@ private ObjectListing getObjectListingChunk(ListObjectsRequest request) {
     @Override
     public ObjectListingChunk getNextChunk() throws IOException {
       if (mResult.isTruncated()) {
+        String nextMarker = mResult.getNextMarker();
+        mRequest.setMarker(nextMarker);
```

Comment:
Could you inline these two calls? `mRequest.setMarker(mResult.getNextMarker())`

Reply:
inline seems good to me. Actually we found this defect exactly in our production environment to list a directory containing 2000+ OSS file. We found it hang forever.

Reply:
change done